.circleci/config.yml

```diff
@@ -93,69 +93,22 @@ jobs:
       
   deploy-doc:
     docker:
-      - image: cimg/node:14.0.0  # npm required for gh-pages
-
+      - image: cimg/python:3.8.12
     steps:
       - checkout
- 
       - add_ssh_keys:
-          # To generate a key run the following in git bash (on Windows):
-          # ssh-keygen -t rsa -m pem -C "ci-build@fairlearn.org"
-          # Do not add a passphrase.
-          # Upload public key as a deploy key to the website repository.
-          # Upload private key to CircleCI as an SSH key at
-          # https://app.circleci.com/settings/project/github/fairlearn/fairlearn#ssh.
-          fingerprints:  # fingerprint of deploy (public) key
-            - "b5:72:60:33:2a:d4:bb:28:0d:36:23:86:74:d3:c6:10"
-
+          fingerprints:
+            - "40:36:a9:4d:ea:9d:d9:94:60:0f:11:b5:fa:6a:b1:5d"
```

Comment:
New key?

Reply:
I wanted to make sure I have a control on everything to test things, can remove it and revert to the old key probably. I'll test.